features/controller_specs/anonymous_controller.feature

```diff
@@ -51,7 +51,7 @@ Feature: anonymous controller
       RSpec.describe ApplicationController, :type => :controller do
         controller do
           def index
-            raise ApplicationController::AccessDenied
+            raise AccessDenied
```

Comment:
Fairly certain this is just a scoping issue not a 1.x thing?

Reply:
Indeed. Decided to keep things as - it's better to be on the safe side.

Reply:
Are you keeping this branch or can it be deleted?

Reply:
Deleted 👍